src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -58,6 +58,8 @@
 import org.jboss.marshalling.Unmarshaller;
 import org.jenkinsci.plugins.workflow.actions.ErrorAction;
 import org.jenkinsci.plugins.workflow.cps.persistence.PersistIn;
+import org.jenkinsci.plugins.workflow.flow.BlockableResume;
```

Comment:
Sorry, where is this defined? I do not see it in https://github.com/jenkinsci/workflow-api-plugin/pull/55. Did you forget to `git add`?

Reply:
Committeed but hadn't been pushed out in the upstream -- you caught me in the midst of pushing out something small, but it's there now.